app/src/main/res/values-nl/strings.xml

```diff
@@ -225,49 +225,49 @@
 
     <!-- Strings displayed only on Google Playstore. Optional, but good to have -->
     <!-- App title has to have less than 50 characters. If you cannot squeeze it, just remove a part of it -->
-    <string name="app_title">Simple Gallery Pro: Photo Manager &amp; Editor</string>
+    <string name="app_title">Eenvoudige Galerij Pro: Foto’s Organiseren &amp; Bewerken</string>
```

Comment:
this is 54 chars long, make it 50 please. "&amp;" counts for 1

Reply:
Done